apps/passport-client/src/dispatch.ts

```diff
@@ -712,7 +713,7 @@ async function doSync(
       console.log("[SYNC] initalized credentialManager", credentialManager);
       const actions =
         await state.subscriptions.pollSubscriptions(credentialManager);
-      console.log("[SYNC] fetched actions", actions);
+      console.log(`[SYNC] fetched ${actions.length} actions`);
```

Comment:
I updated these log statements because they weren't formatted usefully (like `[object Object]`, and if they had been useful they'd have been way too large.